posts/using-napari-for-cosmx-data/index.qmd

```diff
@@ -0,0 +1,525 @@
+are using the two-FOV example above, this would be the folder named `two_fovs_napari_files`. 
+Otherwise, the napari-ready folder is whichever folder containing `images` and `targets.hdf5` (see @sec-napari-ready-files).
+4. napari will ask if you would like to open via the `napari-cosmx` plugin or another
+method. Select the `napari-cosmx` plugin and press okay. 
+
+
+## How to create slides from Raw data {#sec-stitching}
+
+At the time of writing, the process to _create_ napari ready files follows this framework: 
+
+1. Export Raw data from AtoMx SIP (v1.3+) (@sec-export-raw-data)
+2. Launch `napari-cosmx` within napari (@sec-launch-napari-cosmx)
+3. Use the stitch widget to create napari-ready slide from raw data (@sec-stitch-images)
+
+::: {.callout-note}
+One of the main advantages of AtoMx SIP is that the data are stored for you. 
+However, `napari-cosmx` currently requires the raw data downloaded and stored **locally**. Raw data can 
+be quite large (100s of GBs per slide). It is possible
+to store the data on a networked drive but we have noticed that stitching performance
+is slower, depending on your network speed. Storing the data on a high-capacity and
+fast I/O external hard drive may also be an option. 
+:::
+
+
+
+### 1. Export Raw Data {#sec-export-raw-data}
+
+In AtoMx SIP, in the `Study details` panel on the upper left, click `Export` (@fig-atomx-export1). 
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-atomx-export1"
+#| fig-cap: "Click `Export` (available in AtoMx SIP version 1.3+)."
+
+knitr::include_graphics("./figures/export-fig1.png")
+```
+
+Configure your export with the options indicated in @fig-atomx-export2 and click `EXPORT`. If you
+would like to view metadata (optional). You can go ahead and download the Seurat data
+now or as a separate step (_i.e._, Seurat data is not needed for stitching napari
+files). 
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-atomx-export2"
+#| fig-cap: "Example configuration for exporting. For more information on extracting cell-level metadata, see @sec-adding-metadata."
+
+knitr::include_graphics("./figures/export-fig2.png")
+```
+
+When the export is ready, download the data. You can do this over the sftp 
+protocol in a variety of application. Here, I'm using [Cyberduck](https://cyberduck.io/){target="_blank"} but you can use other programs. 
+
+In Cyberduck, click `Open Connection`. In the dropdown menu, select `SFTP` and 
+enter the URL, username, and your (AtoMx) password. Then click `Connect`. Example:
+@fig-atomx-export3.
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-atomx-export3"
+#| fig-cap: "Example configuration for Cyberduck SFTP."
+
+knitr::include_graphics("./figures/export-fig3.png")
+```
+
+Once connected, find the relevant folder, right click, and select `Download As...` (@fig-atomx-export4). 
+Choose the location to on your computer to store the data. You may be able to store
```

Comment:
'location to on your' -> 'location on your'

Reply:
Done: 208857d88eb7bd81da7fd043536a8aad6f51e28b